src/current/v25.4/read-from-standby.md

```diff
@@ -0,0 +1,94 @@
+
+Use this page to understand how the _read from standby_ feature works and how to utilize it.
+
+## How the read from standby feature works
+
+PCR utilizes [cluster virtualization]({% link {{ page.version.version }}/cluster-virtualization-overview.md %}) to separate a cluster's control plane from its data plane. A cluster always has one control plane, called a _system virtual cluster (SystemVC)_, and at least one data plane, called an _App Virtual Cluster (AppVC)_. The standby cluster's SystemVC manages the PCR job and other cluster metadata, and is not used for application queries. All data tables, system tables, and cluster settings in the standby cluster's AppVC are identical to the primary cluster's AppVC. The standby cluster's AppVC itself remains offline during replication.
+
+When using read from standby, applications can read from the standby cluster, but they do not connect directly to the standby cluster's AppVC. Instead, PCR introduces a _reader virtual cluster (ReaderVC)_. The ReaderVC ensures a clean, isolated environment specifically for serving read queries without interfering with replication or system metadata. It reads continuously from the standby cluster's AppVC using internal pointers, providing access to the replicated data while keeping the AppVC offline. The ReaderVC itself only stores a small amount of metadata and no user data, so it is not expected to take up additional storage space.
+
+The standby cluster's ReaderVC has its own system tables and cluster settings. The ReaderVC replicates a subset of system tables, including **Users** and **Roles**, from the AppVC, so that existing primary users can authenticate using the same users and roles as on the primary cluster's AppVC. Other system tables and cluster settings are set to defaults in the ReaderVC.
+
+In the event of failover, the ReaderVC is destroyed.
+
+## Use the read from standby feature
+### Before you begin
+
+Prior to setting up read from standby, ensure that:
+
+- you have already configured PCR between a _primary_ cluster and a _standby_ cluster. For information on configuring PCR, refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}).
+- your CockroachDB version is v24.3 or later. The `read from standby` option is not supported in earlier versions.
+
+### Start a PCR stream with read from standby
+
+To start a PCR stream that allows read access to the standby cluster, use the `CREATE VIRTUAL CLUSTER ... REPLICATION` statement with the `READ VIRTUAL CLUSTER` option:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE VIRTUAL CLUSTER main FROM REPLICATION OF main ON 'postgresql://{connection string to primary}' WITH READ VIRTUAL CLUSTER;
+~~~
+
+### Add read from standby to a PCR stream
+
+To add read from standby capabilities to an existing PCR stream, use the `ALTER VIRTUAL CLUSTER` statement:
```

Comment:
suggest making `ALTER VIRTUAL CLUSTER` a link to that statement's docs: https://www.cockroachlabs.com/docs/v25.3/alter-virtual-cluster.html